SCF-Session-5/N_Queens/NQueenTest.java

```diff
@@ -0,0 +1,31 @@
+package N_Queens;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class NQueenTest
+{
+	NQueenProblem nQueen = new NQueenProblem();
+    
+    	/**
+     	* Test case for n = 4 (solution exists)
+     	*/
+    	@org.junit.Test
```

Comment:
Why `@org.junit.Test` instead of `@test`?

Reply:
done